### PR TITLE
rgw: log errors during "bi list"

### PR DESCRIPTION
**Pull request by ivancich**
Currently errors are displayed on STDERR but do not get into the appropriate log. This duplicates the STDERR messages to the log.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [x] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by thotz — APPROVED** [review]

File: src/rgw/rgw_admin.cc [hunk 1]
```diff
@@ -7670,7 +7670,10 @@ int main(int argc, const char **argv)
 
     int ret = init_bucket(user.get(), tenant, bucket_name, bucket_id, &bucket);
     if (ret < 0) {
-      cerr << "ERROR: could not init bucket: " << cpp_strerror(-ret) << std::endl;
+      ldpp_dout(dpp(), 0) << "ERROR: " << __func__ << ": init_bucket " <<
+	cpp_strerror(-ret) << dendl;
+      cerr << "ERROR: could not init bucket: " << cpp_strerror(-ret) <<
+	std::endl;
```
Comment (cbodley) [inline src/rgw/rgw_admin.cc:7676]: i think it should just be one or the other. log output is already being written to stderr by default, so we can probably just drop the `cerr`?

Comment (ivancich) [reply src/rgw/rgw_admin.cc:7676]: My goal is to make it available in the _client.admin.*.log_ file. So I'll drop the cerr.

Comment (ivancich) [reply src/rgw/rgw_admin.cc:7676]: Fixed.

**Comment by cbodley** [comment]
@ivancich i'm not seeing an updated commit to address my comment, did you maybe forget to push it?

**Review by cbodley — APPROVED** [review]

**Comment by ivancich** [comment]
Teuthology results -- no regressions:
- https://pulpito.ceph.com/ivancich-2023-11-04_15:10:30-rgw-wip-eric-testing-1-distro-default-smithi/
- https://pulpito.ceph.com/ivancich-2023-11-03_20:37:04-rgw-wip-eric-testing-1-distro-default-smithi/

**Comment by ivancich** [comment]
jenkins test api

**Pull request merged**